features/step_definitions/hangout_steps.rb

```diff
@@ -130,3 +141,20 @@
   expect(page).not_to have_content('This event is now live!')
 end
 
+Given(/^that "([^"]*)" went live the previous day$/) do |name|
+  steps %Q{
```

Comment:
we've tried to avoid nested steps due to problems with using the debugger and redirection

I guess we let that go here for the time being, since I think we're all tired of working on this one - would be nice to have a refactoring ticket perhaps ...